### PR TITLE
Call number browse documentation

### PR DESCRIPTION
**Pull request by hectorcorrea**
Formatted section for easier viewing.

### CODE REVIEW

File: docs/call-numbers.md [hunk 1]
```diff
@@ -20,7 +20,26 @@ Expected time: 5.5 - 6 hours.
...
-create a backup of the `tmp/orangelight_name_titles.sorted` file. Count the lines: `wc -l /tmp/orangelight_name_titles.sorted`.
-While on the right box, create a tmux session and run the call number rake task (`cd /opt/orangelight/current && RAILS_ENV=production SOLR_URL=http://lib-solr-prod4.princeton.edu:8983/solr/catalog-production bundle exec rake browse:call_numbers --silent >> /tmp/cron_log.log 2>&1`). It takes some time. When it finishes successfully count the lines in the file `wc -l /tmp/orangelight_name_titles.sorted` to ensure that there is not a big difference with the backup file which is located in `/home/deploy/orangelight_name_titles.sorted` or the most recent file which is located in `/tmp/orangelight_name_titles.sorted`.
-Run `cd /opt/orangelight/current && RAILS_ENV=production SOLR_URL=http://lib-solr8-prod.princeton.edu:8983/solr/catalog-production bundle exec rake browse:load_call_numbers --silent >> /tmp/cron_log.log 2>&1`.
+ssh to `catalog-indexer1`.
+
+Currently the call_numbers task is deployed to run on `catalog-indexer1` check the crontab to confirm. If not then check `catalog-indexer2` and `catalog-indexer3`.
```
Comment (christinach) [inline docs/call-numbers.md:25]: @hectorcorrea it's an opportunity to update the notes in general. Can you update : Currently the call_numbers task is deployed to run on `catalog-indexer1` check the crontab to confirm. If not then check `catalog-indexer2` and `catalog-indexer3`. to => "Currently the call_numbers task is deployed to run on `catalog-indexer1` .https://github.com/pulibrary/orangelight/blob/379f81b953cd76f2e8440f999da02800e6ec0aee/config/schedule.rb#L27. "

File: docs/call-numbers.md [hunk 2]
```diff
@@ -20,7 +20,26 @@ Expected time: 5.5 - 6 hours.
...
+
+Create a backup of the `tmp/orangelight_name_titles.sorted` file.
+
```
Comment (christinach) [inline docs/call-numbers.md:28]: `Create a backup of the `tmp/call_number_browse_s.sorted` file in `/home/deploy/call_number_browse_s.sorted`.

File: docs/call-numbers.md [hunk 3]
```diff
@@ -20,7 +20,26 @@ Expected time: 5.5 - 6 hours.
...
+Count the lines: `wc -l /tmp/orangelight_name_titles.sorted`.
+
```
Comment (christinach) [inline docs/call-numbers.md:30]: Count the lines: `wc -l /tmp/call_number_browse_s.sorted`.

File: docs/call-numbers.md [hunk 4]
```diff
@@ -20,7 +20,26 @@ Expected time: 5.5 - 6 hours.
...
+While on the right box, create a tmux session and run the `call_numbers` rake task:
+
+```
+cd /opt/orangelight/current
+RAILS_ENV=production SOLR_URL=http://lib-solr-prod4.princeton.edu:8983/solr/catalog-production bundle exec rake browse:call_numbers --silent >> /tmp/cron_log.log 2>&1
+```
```
Comment (christinach) [inline docs/call-numbers.md:36]: "... --silent >> /tmp/cron_log_call_numbers.log 2>&1"

File: docs/call-numbers.md [hunk 5]
```diff
@@ -20,7 +20,26 @@ Expected time: 5.5 - 6 hours.
...
+
+It takes some time. When it finishes successfully count the lines in the file `wc -l /tmp/orangelight_name_titles.sorted` to ensure that there is not a big difference with the backup file which is located in `/home/deploy/orangelight_name_titles.sorted` or the most recent file which is located in `/tmp/orangelight_name_titles.sorted`.
```
Comment (christinach) [inline docs/call-numbers.md:38]: It takes around 3 hours. When it finishes successfully count the lines in the file `wc -l /tmp/call_number_browse_s.sorted` to ensure that there is not a big difference with the backup file which is located in `/home/deploy/call_number_browse_s.sorted` or the most recent file which is located in `/tmp/orangelight_name_titles.sorted`.

**Review by christinach — CHANGES_REQUESTED** [review]
@hectorcorrea thank you for checking the documentation. It needs some more updates. I added some comments. Thanks!

**Comment by hectorcorrea** [comment]
@christinach I made the changes that you requested, including renaming the file.

**Pull request merged**